src/liquid/components/ld-pagination/ld-pagination.tsx

```diff
@@ -124,7 +139,7 @@ export class LdPagination {
           part="item focusable"
           size={this.size}
         >
-          {itemNumber}
+          {isDots ? <span class="ld-pagination__dot"></span> : itemNumber}
```

Comment:
```suggestion
          {itemNumber}
```
How about using `.ld-pagination__item--dots ld-button::before` here? Less markup is better in most cases. But if not, please at least use self-closing elements (`<span ... />`).